src/schemas/components/SchemaFormProperty.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <p-label class="schema-form-property" :label="label">
+    <template v-if="description" #description>
+      <div class="schema-form-property__description">
+        <p-markdown-renderer :text="description" class="schema-form-input__markdown" />
+      </div>
+    </template>
+
+    <SchemaFormPropertyInput v-model:value="value" :property="property" />
+  </p-label>
+</template>
+
+<script lang="ts" setup>
+  import { computed } from 'vue'
+  import SchemaFormPropertyInput from '@/schemas/components/SchemaFormPropertyInput.vue'
+  import { SchemaProperty } from '@/schemas/types/schema'
+  import { SchemaValue } from '@/schemas/types/schemaValues'
+
+  const props = defineProps<{
+    property: SchemaProperty,
+    value: SchemaValue,
+    required: boolean,
+  }>()
+
+  const emit = defineEmits<{
+    'update:value': [SchemaValue],
+  }>()
+
+  const value = computed({
+    get() {
+      return props.value
+    },
+    set(value) {
+      emit('update:value', value)
+    },
+  })
+
+  const label = computed(() => {
+    const title = props.property.title ?? ''
+
+    if (!props.required) {
+      return `${title} (Optional)`.trim()
+    }
+
+    return title
+  })
+
+  const description = computed(() => {
+    const { description = '' } = props.property
+    const descriptionWithNewlinesRemoved = description.replace(/\n/g, ' ')
+
+    return descriptionWithNewlinesRemoved
```

Comment:
What's going on here? This doesn't feel like it should be necessary

Reply:
Fixes this
<img width="558" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/6200442/18bf2602-aafb-46d2-a251-206166c8fbd7">
To display this
<img width="571" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/6200442/806be656-cd46-4848-ad4b-0e63c97d8776">

These descriptions come directly from code comments. And those code comments usually have line breaks in them to format them at a certain line length. But then in the ui they break in weird places and have gaps in them like there are multiple paragraphs.

Reply:
Ah I see, ok that makes sense. How would you feel about replacing the regex with this:
```
/\n(?!\n)/g
```

That'll do a negative lookahead to make sure we're not incorrectly stripping newline characters that would mark paragraph boundaries